docs/source/whats_new.rst

```diff
@@ -8,6 +8,14 @@ What's New
 Current
 -------
 
+Bug
+~~~
+
+- In find_noisy_channels the signal detrend was accidentally undone which destabilized the RANSAC.
+- The detrend is now done over the NoisyChannels.raw_mne object to avoid this and to force that any signal in there is detrended like in matlab's prep.
+- Included a boolean which indicates if detrend should be done internally or not for the use of find_noisy_channels in reference.py
```

Comment:
this particular line should go under "changelog", because it's a feature that you added

Reply:
ok, taking care of this now

Reply:
Im confused, should i make a new changelog since the current changelog is from a previous request?

Reply:
No, `changelog` is just the heading, under that heading we have an unordered list ... and every item in that list is one change (i.e., a feature or something that is NOT a bugfix, or an API change) see for example here: https://raw.githubusercontent.com/mne-tools/mne-bids/master/doc/whats_new.rst

Reply:
you can also add a link to your pull request using something like:

`by `Yorguin Mantilla`_ (`#29 <https://github.com/sappelhoff/pyprep/pull/9>`_)`

Reply:
Thanks, looking at the example I think I got it.